### ISSUE TITLE
Talkback focus is moving to the invisible "Keyboard control" section present under "Transcript section.

### ISSUE THREAD

**Issue by pgopinadhu** [microsoft/BotFramework-WebChat#4965]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Talkback users will not be able to know there is section present under "Transcript" section to know the keyboard controls. There is a keyboard controls popup after voiceover/talkback enabled on the phone. 

### What browsers and screen readers do this issue affect?

iOS/iPadOS: Safari with VoiceOver, Android: Chrome with TalkBack, Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

1. Launch URL: [https://aurorabapenv9c3fa.crm10.dynamics.com/](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faurorabapenv9c3fa.crm10.dynamics.com%2f&data=05%7c01%7cv-saddepalli%40microsoft.com%7ce7f2f9a57f1e43ee7fdf08dbb062c615%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638297710577355999%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=CdDCi5Zs3cWZonoUZ2oROBYZo2%2BByuFjuXx3t5HadCE%3D&reserved=0) and log in with valid credentials. 

2. Swipe to "Customer Service Hub" app and press enter. 

3. Swipe to "Sitemap" app and press enter​. 

4. Swipe to "Service" and press enter then choose "Cases". 

5. Select any case from the grid. 

6. Swipe to timeline section. 

7. Swipe to the records Under timeline section 

8. Swipe to "Open record" options in the record and press enter. 

9. Swipe to "Transcript" section. 

10. Now verify the issue talkback focus is moving to the invisible "Keyboard control" section present under "Transcript section.

### What do you expect?

Screen reader should not read the text which is not visible on the screen or "Keyboard control" section present under "Transcript section should be visible to the users. 

### What actually happened?

Talkback focus is moving to the invisible "Keyboard control" section present under "Transcript section. It still tries to read the text on the hidden popup.

**Environment Details:**
Application Name: Dynamics 365 - Enterprise Service 

URL: [https://aurorabapenv9c3fa.crm10.dynamics.com/](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faurorabapenv9c3fa.crm10.dynamics.com%2f&data=05%7c01%7cv-saddepalli%40microsoft.com%7ce7f2f9a57f1e43ee7fdf08dbb062c615%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638297710577355999%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=CdDCi5Zs3cWZonoUZ2oROBYZo2%2BByuFjuXx3t5HadCE%3D&reserved=0) 

Android:  OnePlus Nord CE 2 

Android version 13

### Do you have any screenshots or recordings to repro the issue?

https://microsoftapc-my.sharepoint.com/personal/v-tharika_microsoft_com/_layouts/15/stream.aspx?id=%2Fpersonal%2Fv%2Dtharika%5Fmicrosoft%5Fcom%2FDocuments%2F3593659%5FA11y%5FDynamics%20365%20%2D%20Enterprise%20Service%5FView%20email%5FTo%20view%20email%5FScreen%20Reader%20%281%29%2Emp4&ct=1700721179194&or=Outlook%2DBody&cid=8355DEA6%2DA146%2D4804%2DBCFE%2D0C08F1910EB6&ga=1&LOF=1&referrer=StreamWebApp%2EWeb&referrerScenario=AddressBarCopied%2Eview
<img width="222" alt="img1" src="https://github.com/microsoft/BotFramework-WebChat/assets/120019572/152dce1f-1b09-47fb-a001-d203ed1ab596">

### MAS reference

https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt

### Additional context

Bug Copied from TCS Team: [BUG 3593659](https://dev.azure.com/dynamicscrm/OneCRM/_workitems/edit/3593659)

**Comment by compulim** [comment]
I will talk to accessibility team tomorrow about this issue.

**Comment by Kirank915** [comment]
@compulim do you have any update on this issue?

**Comment by kattepalli** [comment]
@compulim As verified issue is still repro and do you have any update on this issue?
